src/main/java/org/apache/maven/plugins/gpg/SignDeployedMojo.java

```diff
@@ -106,11 +105,15 @@ public class SignDeployedMojo extends AbstractGpgMojo {
     @Parameter(property = "artifacts")
     private String artifacts;
 
-    @Component
-    private RepositorySystem repositorySystem;
+    private final RepositorySystem repositorySystem;
+
+    private final Map<String, ArtifactCollectorSPI> artifactCollectors;
```

Comment:
why we need it?

Reply:
I'm not 100% sure. It is used by the existing code.

Reply:
but `artifactCollectors` looks like new variable added by You

Reply:
No. it was just moved a little bit. It's on line 113 in the original.